openedx/adg/lms/webinars/models.py

```diff
@@ -99,6 +114,21 @@ def clean(self):
         if errors:
             raise ValidationError(errors)
 
+    def delete(self, *args, **kwargs):  # pylint: disable=arguments-differ, unused-argument
+        if self.status == Webinar.UPCOMING:
+            send_cancellation_emails_for_given_webinars([self])
+        self.status = self.CANCELLED
+        self.save()
+
+
+class CancelledWebinar(Webinar):
```

Comment:
Please create and add migration for this model

Reply:
I didn't realize proxy models created a migration. Just read up on this. I'll add it as well.
Great catch!